aiopg/pool.py

```diff
@@ -17,20 +17,21 @@
 
 
 def create_pool(dsn=None, *, minsize=1, maxsize=10,
-                loop=None, timeout=TIMEOUT,
+                loop=None, timeout=TIMEOUT, pool_recycle=-1,
```

Comment:
We need to add check for `pool_recycle` validation, like we do for  `minsize`, also what do you think about making default value `None` or large number, say 3600 seconds?

Reply:
@jettify 
When you have database on localhost - looks like you don't need connection recycling. I wasn't able to reproduce this issue on localhost. So, large values weren't needed by default.

What about `-1` - I use this value only for compatibility with SQLAlchemy. Why did they use `-1` - I don't know.

Reply:
agreed